src/org/labkey/test/pages/experiment/InsertDataClassPage.java

```diff
@@ -19,9 +19,11 @@
 import org.labkey.test.WebDriverWrapper;
```

Comment:
Should this class/file really be named CreateDataClassPage? It would make the naming more consistent with the CreateSampleSetPage.

Reply:
You'll have to take that up with `InsertDataClassAction`. I don't know why or when we prefer insert vs create vs import much of the time.